knowledge/banking/DORA/qna.yaml

```diff
@@ -0,0 +1,98 @@
+  - question: What is a requirement for Operational Resilience Testing under DORA?
+    answer: |
+      Physical security reviews is required for Operational Resilience Testing under DORA
+ - question: What is the position in DORA in regards to Open Source?
+    answer: |
+      Open source code and source code reviews is needed for Operational Resilience Testing under DORA.
+ - question: How can Financial Entities implement a Risk Management framework?
+    answer: |
+      They can adopt a governance and control framework.
+ - question: What are the potential financial impacts to ICT providers in case of DORA incompliance?
+    answer: |
+      DORA allows lead overseers to levy fines on ICT providers amounting to 1% of the provider's average daily worldwide
+      turnover in the previous business year. Providers can be fined every day for up to six months until they achieve compliance.
+ - question: What is a critical business function of a financial institution that falls under DORA?
+    answer: |
+      Completing bank checking account transactions is considered a critical business function?
+ - question: Is a loans application considered a critical business function for a financial institution in DORA?
+    answer: |
+      Accepting loan applications in a timely manner is a critical business function for a bank.
+ - question: Do all Cloud service providers fall under DORA?
+    answer: |
+      Cloud services providers that serve EU banks but are based outside the EU fall under DORA.
+ - question: Are all ICT providers subject to DORA?
+    answer: |
+      Data modeling services firms providing analytical models to EU insurers are subject to DORA.
+ - question: Is DORA relevant for all IT service providers?
+    answer: |
+      External IT service firms providing customer identity services to EU banks need to comply with DORA
+ - question: When will DORA be enforced?
+    answer: |
+      The Digital Operational Resilience Act (DORA) is a EU regulation that entered into force on 16 January
+      2023 and will apply as of 17 January 2025.
+ - question: Which are the 4 domains of DORA?
+    answer: |
+      DORA establishes technical requirements for financial entities and ICT providers across four domains.
+      These domains are ICT risk management and governance, incident response and reporting, digital
+      operational resilience testing, and third-party risk management.
+      While information sharing is encouraged, it is not mandatory.
+      The requirements will be enforced proportionately, meaning smaller entities will not be held to the
+      same standards as major financial institutions. Although the Regulatory Technical Standards (RTSs)
+      and Implementing Technical Standards (ITSs) for each domain are still under development,
+      the existing DORA legislation provides some insight into the general requirements.
+
+#TODO
+document:
+  repo: https://github.com/juliadenham/oscars2024_knowledge.git
```

Comment:
If you are making a knowledge contribution, you should then be providing your knowledge markdown files in a git repo. This is the link to some other knowledge contribution's git repo holding markdown files.